api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/WorksQuery.scala

```diff
@@ -34,17 +38,25 @@ case object WorksMultiMatcher {
             (Some(1000), "data.imageData.id.otherIdentifiers.value"),
           ).map(f => FieldWithOptionalBoost(f._2, f._1.map(_.toDouble)))
         ),
-        prefixQuery("data.title.keyword", q).boost(1000),
         MultiMatchQuery(
           q,
-          `type` = Some(CROSS_FIELDS),
-          operator = Some(AND),
+          `type` = Some(BEST_FIELDS),
+          operator = Some(OR),
           fields = Seq(
-            (Some(1000), "data.contributors.agent.label"),
             (Some(100), "data.title"),
             (Some(100), "data.title.english"),
             (Some(100), "data.title.shingles"),
             (Some(100), "data.alternativeTitles"),
+            (None, "data.lettering"),
```

Comment:
Why did lettering get in here?

Reply:
lettering is another field where we see multiple languages used, so imo we should be applying fuzz to it to catch transliteration issues

Reply:
Assuming this is language type fields, maybe we could look at this implementation once we search multiple language fields?

Reply:
I wasn't very clear, I wonder if we could leave this implementation out until we actually search across multiple language fields.

Reply:
sorry, i should have specified - lettering is a place where we also see _transliteration_ issues (not just multiple languages), which is what the fuzzy match is intended to solve

Reply:
What is the point of the `BEST_FIELDS` then?